packages/astro/src/client/react/uiComponents.tsx

```diff
@@ -0,0 +1,134 @@
+import type {
```

Comment:
❓ This file is copy pasted from `@clerk/clerk-react` right?

Reply:
Yes, but `withClerk` is a custom implementation.